github-notify.go

```diff
@@ -64,13 +66,80 @@ func main() {
 	for {
 		println("Checking notifications...")
 		notes := retrieveNotifications(client)
-		// if there are mentions push one to pushbullet
-		if len(notes) > 0 {
-			fmt.Printf("Found %d mentions; pushing one.\n", len(notes))
+		notes = reverse(notes);
+		polling := mapToID(notes);
```

Comment:
Wouldn't it be better to just store the whole notification instead of the id of the issue? What do we gain by storing just the id? Memory shouldn't be an issue at this scale.